package.json

```
@@ -53,6 +54,7 @@
         "lint-staged": "15.2.2",
         "typescript": "5.2.2",
         "vite": "5.2.6",
+        "vite-bundle-visualizer": "^1.2.1",
```

```suggestion
        "vite-bundle-visualizer": "1.2.1",
```

lock dependency to a version